app/assets/stylesheets/_audit.scss

```diff
@@ -24,22 +24,19 @@ $next-link-colour: #000000;
   margin-bottom: 1.5em;
 }
 
-.question {
-  &.is-there-content-out-of-date {
-    margin-top: 4em;
-  }
+.t-reformat {
+  margin-bottom: 4em;
 }
 
-.free-text-question {
-  &.redirect-or-combine-with-url {
-    textarea {
-      height: 2em;
-    }
-  }
+.t-redirect-urls,
```

Comment:
What does the prefix `.t-` signify?

Reply:
I don't know - it was already a thing but as I'm not able to answer your question I've removed it.